src/main/java/se/iths/jakartaeelab1/service/PersonService.java

```diff
@@ -8,13 +8,15 @@
 import se.iths.jakartaeelab1.entity.Person;
 import se.iths.jakartaeelab1.repository.PersonRepository;
 
-import java.util.List;
 import java.util.UUID;
 
 @ApplicationScoped
 public class PersonService {
     PersonRepository personRepository;
 
+    public PersonService() {
+    }
+
```

Comment:
Good catch, forgot about this completetly.

Reply:
It didnt work without it 😸 gave me the red carpet of errors